homeassistant/components/homekit/accessories.py

```diff
@@ -88,8 +109,28 @@ def update_state_callback(self, entity_id=None, old_state=None,
         _LOGGER.debug('New_state: %s', new_state)
         if new_state is None:
             return
+        self.update_battery_level(new_state)
         self.update_state(new_state)
 
+    def update_battery_level(self, new_state):
+        """Update battery service if available."""
+        battery_level = convert_to_float(new_state.attributes
+                                         .get(ATTR_BATTERY_LEVEL))
+        if battery_level is not None:
```

Comment:
You could reduce the indent level for the rest of the function by using
```py
if battery_level is None:
    return

if battery_level <=1:
    # an so on
```

Reply:
Good catch, done.